### ISSUE TITLE
Setting more options

### ISSUE THREAD

**Issue by tinyweasel** [mfcollins3/ember-datatables#1]
I've created a datatable that inherits from the ember-datatables component in the node_modules folder. Is is possible to set more options on it, such as the ones in https://datatables.net/reference/option/ without changing the parent component?

**Comment by mfcollins3** [comment]
Yes. My apologies. I've had some critical deadlines for my day job and haven't been able to post any new updates, but I can work on it tonight and over the weekend for you. Which options do you specifically need support for so that I can prioritize them for you?

Thank you for giving my add-on a try.

**Comment by tinyweasel** [comment]
Hi again! Sorry for reply taking ages. I'm still interested in this issue though; also, is there a way to access the row details to set properties on them? I'm thinking specifically to make a certain field into a href. Thanks for your help!

**Comment by mfcollins3** [comment]
No worries. I was traveling last week. I added more of the options to the component. I'll try to finish all of them this weekend and will look into the row details thing. I'll see if I can get an updated version with everything included for you by the end of the weekend.

**Comment by tinyweasel** [comment]
Great, thanks so much!

**Comment by tinyweasel** [comment]
Hey, did you manage to have a look at this? I'm interesting in making some of the rows correspond to a hasMany relationship, so their property would be an array of values.
